docker-compose.yml

```diff
@@ -1,159 +1,189 @@
-version: '2'

```

Comment:
Why are we deleting these lines?
Any pupose?

Reply:
These are not removed, just check contents. Have removed ^M chars